lib/datawow/connection.rb

```diff
@@ -4,13 +4,14 @@
 module Datawow
   # :nodoc:
   class Connection
-    def initialize(model = nil)
+    def initialize(model:, version_api: 'v1')
       @model = model.to_sym
```

Comment:
From the changes, `model` is already be `symbol`. Thus no need to symbolize again.